jpa/spec-tests/pom.xml

```diff
@@ -89,4 +89,127 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>hibernate</id>
+            <properties>
+                <hibernate.version>7.0.0-SNAPSHOT</hibernate.version>
```

Comment:
avoid SNAPSHOTs, if one needs to use it, (s)he can pass the version through the property (should also be fixed in `bin/pom.xml`)